tools/dcos_login.py

```diff
@@ -202,16 +113,7 @@ def ignore_empty(envvar, default):
 
     dcos_login_username = ignore_empty("DCOS_LOGIN_USERNAME", __CLI_LOGIN_EE_USERNAME)
     dcos_login_password = ignore_empty("DCOS_LOGIN_PASSWORD", __CLI_LOGIN_EE_PASSWORD)
-    dcos_enterprise = ignore_empty("DCOS_ENTERPRISE", "true").lower() == "true"
-    dcos_acs_token = os.environ.get("DCOS_ACS_TOKEN")
-    if not dcos_acs_token:
-        dcos_acs_token = login(
-            dcosurl=cluster_url,
-            username=dcos_login_username,
-            password=dcos_login_password,
-            is_enterprise=dcos_enterprise,
-        )
-    configure_cli(dcosurl=cluster_url, token=dcos_acs_token)
+    subprocess.run(["dcos", "cluster", "setup", cluster_url, "--username", dcos_login_username, "--password", dcos_login_password])
```

Comment:
One thing to note is that this doesn't handle the case where we run this against a cluster that is already attached / setup.

Reply:
Under what conditions these tests can be run? With the new CLI the cluster setup command became idempotent, re-running it will just overwrite the previous config if any.

Otherwise it is also possible to run `dcos cluster list --attached --json` to see if there is any attached cluster and skip the setup based on this.

Reply:
Thanks for the info. That will simplify things a lot. Then we only have to handle 1.9 clusters as an edge case.